docs/guide/explanation/the-deployment-process.mdx

```diff
@@ -0,0 +1,53 @@
+# The deployment process
+Deployment begins via one of Zephyr Cloud's bundler plugins.
```

Comment:
I'm not a fan of the word via, I believe its something something that non native english speakers may struggle with.

Reply:
Good spot, and agreed.